src/components/Title/Title.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+class Title extends React.Component {
+    render(){
+        return (
+            <div className="title-container">
+                   <h1 className="Title">Hot Dog</h1>
```

Comment:
If we wanted to make this a truly reusable component, I think it would be worth passing in the title and instruction as props

Reply:
Doesn't look like this has been looked at?

Reply:
on it